Carbon.MassTransit/IServiceCollectionConfiguratorExtensions.cs

```diff
@@ -445,21 +444,69 @@ public static void AddAsyncRequestResponsePatternForResponder(this IServiceColle
         /// </summary>
         /// <param name="cfg"></param>
         /// <param name="configuration">Configuration</param>
+        /// <remarks>Currently only works with RabbitMQ environments.</remarks>
         public static void AddAsHighAvailableQueue(this IRabbitMqReceiveEndpointConfigurator cfg,
                                       IConfiguration configuration)
+        {
+            var massTransitSettings = configuration.GetSection("MassTransit").Get<MassTransitSettings>();
+            if (massTransitSettings == null)
```

Comment:
No need to check, already checked previously and initially while adding masstransit